### ISSUE TITLE
Weighted Loss Function feature requirement

### ISSUE THREAD

**Issue by kdgutier** [awslabs/keras-apache-mxnet#267]
Hi I am using the keras-mxnet api, I am finding it hard to declare a weighted loss function.

I know it cannot be registered with mxnet backend.
There is no available example here https://github.com/awslabs/keras-apache-mxnet/blob/master/keras/losses.py#L77 for a weighted loss.

Is there a keras backend operation can help to do a basic matrix multiplication on the batch dimension.
